source/developer-guides/ias.rst

```diff
@@ -1,24 +1,24 @@
 IAS Image
 -----------
 
-Booting quickly and securely is highly desirable, and in many use cases, mandatory. |SPN| is designed to boot OS by directly loading and verifying a kernel image in a raw format known as IAS image.
+Booting quickly and securely is highly desirable, and in many use cases, mandatory. |SPN| is designed to boot OS by directly loading and verifying a kernel image in a raw format known as IAS image. |SPN| supports |Multiboot Protocol| compliant boot images encapsulated in an IAS image.

```

Comment:
"Designed to boot OS" should be "Designed to boot **an** OS"

I think the phrasing "raw format" here is confusing and should really be "container format" or something, since IAS Image format images contain multiple things [or maybe I don't understand what *raw* means in this context?]